ajax/libs/piwik/package.json

```diff
@@ -12,12 +12,20 @@
   "maintainers": [
     {
       "name": "The Piwik Team",
-      "url": "http://piwik.org/the-piwik-team/"
+      "url": "http://piwik.org/team/"
```

Comment:
The origin url still works, I recommend not to touch it here, or should better use another commit to touch it.